tests/langchain/test_langchain_model_export.py

```diff
@@ -174,6 +174,81 @@ def create_openai_llmagent(return_intermediate_steps=False):
     )
 
 
+def create_uc_tools(
+    monkeypatch, warehouse_id, expected_catalog_name, expected_schema_name, functions
+):
+    try:
+        from langchain_community.tools.databricks import UCFunctionToolkit
+    except Exception:
+        return []
```

Comment:
Why do we need this try catch? I think you're guarding the import by only running the tests on langchain version  >= 0.2.0

Reply:
For `test_databricks_dependency_extraction_from_agent_chain` we are restricting it to >= 0.1.0 which does not have uc function toolkit. The idea is for >= 0.1.0 we support Agent Executors without UCFunctions, they can still use retriever tool. For >= 0.2.0 we support UCFunctions, Agent Executors and Langgraph

Reply:
I see, can we remove `if len(uc_function_tools) > 0:` check in the `test_databricks_dependency_extraction_from_langgraph_agent` test to make sure the tools are always created and captured?